src/style.css

```diff
@@ -0,0 +1,105 @@
+body {
+  font-family: Roboto;
```

Comment:
In this task you should drop default body margins.

Reply:
> In this task you should drop default body margins.
l use reset style connected from the outside